stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -395,6 +395,13 @@ impl StacksHeaderInfo {
     pub fn is_nakamoto_block(&self) -> bool {
         matches!(self.anchored_header, StacksBlockHeaderTypes::Nakamoto(_))
     }
+
+    pub fn header_type(&self) -> &str {
```

Comment:
Maybe `header_type_name` would be a better function name?  It's returning a `&str`, not a bespoke type.